test/nodes/quantum-register_spec.js

```diff
@@ -23,9 +23,9 @@ describe('QuantumRegisterNode', function() {
     testUtil.isLoaded(quantumRegisterNode, 'quantum-register', done);
   });
 
-  it('execute command', function(done) {
-    // Test is disabled until issue #87 is fixed
+  it('execute quantum circuit initilization command if it is the last register node connected', function(done) {
```

Comment:
```suggestion
  it('execute quantum circuit command if last register is connected', function(done) {
```
Again, I think this test name is a bit verbose and can be shortened.